### ISSUE TITLE
Out of place sentence in chapter 6

### ISSUE THREAD

**Issue by LucianU** [AllenDowney/ThinkStats2#30]
On page 92, there is the following code:

```python
sample = [random.gauss(mean, std) for i in range(500)]
sample_pdf = thinkstats2.EstimatedPdf(sample)
thinkplot.Pdf(sample_pdf, label='sample KDE')
```
Below, there is a sentence starting with

>  `pmf` is a Pmf object that ...

However, `pmf` doesn't appear anywhere on the page.

**Comment by pglezen** [comment]
This may be an older version of the text.  My version (in which the snippet appears on page 68 of the PDF document) has the correct phrase

> sample_pdf is a Pdf object that ...

So your correct in your observation; just a bit behind on the latest version of the text.

**Comment by AllenDowney** [comment]
Thanks, Paul.  Lucien, does that resolve the issue?

**Comment by LucianU** [comment]
The sentence still talks about `pmf`, which doesn't appear at all in the 3 lines of code.

**Comment by AllenDowney** [comment]
Ah, got it.  Resolved with commit 6626811

Thanks!
